cmd/promobot-generate-manifest/main.go

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"golang.org/x/xerrors"
+	"k8s.io/klog"
+	"sigs.k8s.io/k8s-container-image-promoter/pkg/cmd"
+	"sigs.k8s.io/yaml"
+)
+
+func main() {
+	ctx := context.Background()
+
+	if err := run(ctx); err != nil {
+		fmt.Fprintf(os.Stderr, "%v\n", err)
+		// nolint[gomnd]
+		os.Exit(1)
+	}
+	os.Exit(0)
+}
+
+// nolint[lll]
+func run(ctx context.Context) error {
+	klog.InitFlags(nil)
+
+	src := ""
+	flag.StringVar(
+		&src,
+		"src",
+		src,
+		"the base directory to copy from")
+
+	flag.Parse()
+
+	if src == "" {
+		return xerrors.New("must specify --src")
+	}
+
+	s, err := filepath.Abs(src)
+	if err != nil {
+		return xerrors.Errorf("cannot resolve %q to absolute path: %w", src, err)
+	}
+	src = s
+
+	var opt cmd.GenerateManifestOptions
+	opt.BaseDir = src
```

Comment:
Maybe just do `opt.BaseDir = s` and delete line 64?

Reply:
Good call - done